assets/src/edit-story/components/header/title.js

```diff
@@ -49,6 +51,14 @@ const Input = styled.input`
   text-overflow: ellipsis;
 `;
 
+const HighlightContainer = styled.div`
+  display: inline-block;
+  ${({ focusContainerCss }) => focusContainerCss}
+  ${({ focusContainerSelector }) => focusContainerSelector} {
+    border-radius: 4px;
```

Comment:
not sure if we're integrating the design system styling yet, but if we are we could use `theme.borders.radius.small` here

Reply:
I'll import the ds theme so we'll see it when it's integrated